### DISCUSSION TITLE
Create view for all documents having an ASN

### DISCUSSION THREAD

**Discussion by steilerDev** [jonaswinkler/paperless-ng#1210] [Support] [answered]
Hi All,

love the tool and just migrated all of my physical documents (~2500 documents) to paperless-ng, amazing system!

I am now building my remaining physical archive and I would like to have a view, showing me all documents having an ASN. Unfortunately in the "Saved view" admin panel, I can only see filtering by "does not have ASN", is there a way to achieve "has ASN" that I am overlooking?

Thanks!

**Comment by bergfex** [comment, marked as answer]
Hi steilerDev,

just enter `asn:[0 to]` in the advanced search, or the range you want to search. Make sure there is no space between the "to" and the closing square bracket.

See also the [documentation](https://paperless-ng.readthedocs.io/en/latest/usage_overview.html#searching) and also the link to the [Whoosh query language](https://whoosh.readthedocs.io/en/latest/querylang.html)

**Reply by steilerDev** [reply]
Works like a charm, thank you! :)

**Reply by bergfex** [reply]
Nice to hear. Thanks for the feedback

**Reply by MichaelVessia** [reply]
I had the same exact problem and this helped me as well. Thanks!
